test/e2e/tag/tag.spec.ts

```diff
@@ -2,17 +2,103 @@ import { test, expect, Page } from '@playwright/test';
 import { StoryBookPage } from '../storybookPage';
 
 const getPage = (page: Page) =>
-  new StoryBookPage(page, '?path=/story/next-tag--interactive');
+  new StoryBookPage(page, '?path=/story/next-tag--default');
 
-test('tag - standard size', async ({ page }) => {
-  const tagPage = getPage(page);
-  await tagPage.goto();
-  await expect(tagPage.container).toHaveScreenshot('tag-standard-size.png');
+const getRemoveableTagPage = (page: Page) =>
+  new StoryBookPage(page, '?path=/story/next-tag--removeable');
+
+test.describe('default tag', () => {
```

Comment:
can you help to do the same as the comment here: https://github.com/glints-dev/glints-aries/pull/736#discussion_r1091595232

Reply:
remove the describe test suite

now everything's flat